website/static/js/profile.js

```diff
@@ -577,6 +609,41 @@ var ListViewModel = function(ContentModel, urls, modes) {
         }
         return true;
     });
+    
+
+    self.institutionsEmpty = ko.computed(function() {
+
+            for (var i=0; i<self.contents().length; i++) {
+                if (self.contents()[i].institutionEmpty()) { 
+                    return true;
+                }
+            }
+ 
+        return false;
+                
+    });
+
+    
+    self.extraFieldsEmpty = ko.computed(function() {
+        var i;
```

Comment:
jshint will flag using the same variable twice in a function.  To get around it, I could use a different variable 

```
else if (urls.crud === '/api/v1/settings/schools/') {
            for (var j=0; j<self.contents().length; j++) {
                if (self.contents()[j].department() === '' && self.contents()[j].degree() === '') {
                    return true;
                }
            }
        }
```

or I could put  
```
/* jshint shadow:true */
```
in tell jshint to ignore it.  Any thoughts? @lyndsysimon or @sloria